[inductor] Disable mixed-order reduction for cpp-wrapper

desertfire opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #169859

Summary: Fixes https://github.com/pytorch/pytorch/issues/169664. A temporary fix to bring CI to back to green. Will work on another PR to add proper cpp-wrapper codegen support for mixed-order reduction.

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @kadeng @muchulee8 @amjames @chauhang @aakhundov @coconutruben @jataylo @mlazos

desertfire committed:
[inductor] Disable mixed-order reduction for cpp-wrapper

Summary: Fixes https://github.com/pytorch/pytorch/issues/169664. A temporary fix to bring CI to back to green. Will work on another PR to add proper cpp-wrapper codegen support for mixed-order reduction.

[ghstack-poisoned]

shunting314 reviewed (commented):
Thanks for fixing it! Just curious, do we still run cpp-wrapper tests in CI? Since CI was green when I landed multiple PRs related to mix-order reduction.

desertfire committed:
Update on "[inductor] Disable mixed-order reduction for cpp-wrapper"


Summary: Fixes https://github.com/pytorch/pytorch/issues/169664. A temporary fix to bring CI to back to green. Will work on another PR to add proper cpp-wrapper codegen support for mixed-order reduction.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo mlazos

[ghstack-poisoned]
